server/tests/common/rpc.py

```diff
@@ -31,12 +31,25 @@ async def ping(self, ping: str) -> anonymous_cmds.latest.ping.Rep:
         raw_rep = await self._do_request(req.dump())
         return anonymous_cmds.latest.ping.Rep.load(raw_rep)
 
-    async def pki_enrollment_info(
-        self, enrollment_id: EnrollmentID
-    ) -> anonymous_cmds.latest.pki_enrollment_info.Rep:
-        req = anonymous_cmds.latest.pki_enrollment_info.Req(enrollment_id=enrollment_id)
+    async def pki_enrollment_submit(
```

Comment:
A specific reason why those functions `pki_enrollment_info` and `pki_enrollment_submit` were inverted in position ?

Reply:
NVM: It's because it's auto-generated

https://github.com/Scille/parsec-cloud/blob/4687a9109461f2b667d25c7df668a96d3dcae49b/server/tests/common/rpc.py#L3

Looks like someone forgot to sort the method before